test/ci.py

```diff
@@ -285,4 +285,4 @@ def test_install_scripts_prefix_override(self):
         pip3_regex = re.compile('pip3 install')
         pip3_install_scripts = pip_install_scripts.replace('pip ', 'pip3 ')
         expected_jenkinsfile = pip3_regex.sub(pip3_install_scripts, EXPECTED_JENKINSFILE_PIP3_INSTALL_TOX)
```

Comment:
this line is no longer needed?

Reply:
indeed, cleaned up in https://github.com/hpcugent/vsc-install/pull/172